README.md

```diff
@@ -2,77 +2,107 @@
 
-This is where PyDP comes in. PyDP is a Python wrapper for Google's [Differential Privacy](https://github.com/google/differential-privacy) project. The library provides a set of ε-differentially private algorithms, which can be used to produce aggregate statistics over numeric data sets containing private or sensitive information. Thus, PyDP is helping us achieve better privacy.
+**Things to remember about PyDP:**
 
-**Things to remember about PyDP :**
-- :rocket: Features differentially private algorithms including: BoundedMean, BoundedSum, Max, Count Above, Percentile, Min, Median, etc.  
-- All the computation methods mentioned above use Laplace noise only. (Other noise mechanisms will be added soon... :smiley:)
-- :fire: Currently supports Linux and OSX. (Windows coming real soon... :smiley:)
-- :star: Supports all the Python 3+ versions.
+- :rocket: Features differentially private algorithms including: BoundedMean,
+BoundedSum, Max, Count Above, Percentile, Min, Median, etc.
+  - All the computation methods mentioned above use Laplace noise only (other
+noise mechanisms will be added soon! :smiley:)
+- :fire: Currently supports Linux and macOS (Windows support coming soon
+:smiley:)
+- :star: Use Python 3.x.
 
 ## Installation
-Use the package manager [pip](https://pip.pypa.io/en/stable/) to install PyDP.
+
+To install PyDP, use the [PiPy](https://pip.pypa.io/en/stable/) package manager:
 
 ```bash
 pip install python-dp
 ```
 
-## Usage
-Refer to [this example](https://github.com/OpenMined/PyDP/tree/dev/examples) to understand PyDP library usage.
-
-For usage via code explanation, refer to [Jupyer Notebook](https://github.com/OpenMined/PyDP/blob/dev/examples/carrots_demo/carrots_demo.ipynb) or [Python file](https://github.com/OpenMined/PyDP/blob/dev/examples/carrots_demo/carrots.py) for carrot demo.
+(If you have `pip3` separately for Python 3.x, use `pip3 install python-dp`.)
 
-Documentation can be found [here](https://openmined.github.io/PyDP/readme.html).
+## Examples
 
-A sample of usage can be found below:
+Refer to the
+[curated list](https://github.com/OpenMined/PyDP/tree/dev/examples)
+of tutorials and sample code to learn more about the PyDP library.
 
-```python
-import pydp as dp # imports the DP library
-from pydp.algorithms.laplacian import BoundedMean
+You can also get started with
+[an introduction to PyDP](https://github.com/OpenMined/PyDP/blob/dev/examples/carrots_demo/carrots_demo.ipynb)
+(a Jupyter notebook) and
+[the carrots demo](https://github.com/OpenMined/PyDP/blob/dev/examples/carrots_demo/carrots.py)
+(a Python file).
 
+Example: calculate the Bounded Mean
 
-# To calculate the Bounded Mean
-# epsilon is a number between 0 and 1 denoting privacy threshold
-# It measures the acceptable loss of privacy (with 0 meaning no loss is acceptable)
-# If both the lower and upper bounds are specified, 
-# x = dp.BoundedMean(epsilon: double, lower: int, upper: int)
+```python
+# Import PyDP
+import pydp as dp
+
```

Comment:
Also, the import is not changed to 
`from pydp.algorithms.laplacian import BoundedMean(`

Reply:
and the subsequent calling of BoundedMean

Reply:
Yes! I was using the older version of README. Thanks!